pkgs/sagemath-objects/MANIFEST.in

```diff
@@ -83,6 +83,7 @@ graft sage/libs/gmp
 
 ## FIXME: Needed for doctesting
 include sage/misc/misc.*              # walltime, cputime used in sage.doctest
+include sage/misc/timing.p*
```

Comment:
the comment `# walltime, cputime used in sage.doctest` should be moved on this line

Reply:
Thanks, fixed in af7bcf6